<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultForm</class>
 <widget class="QDialog" name="ResultForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>390</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>0</y>
     <width>291</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>244</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="proteinL">
      <property name="text">
       <string>Протеин:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="protein"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="allTimeL">
      <property name="text">
       <string>Время всех запусков</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="allTime">
      <property name="text">
       <string>time</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="averageTimeL">
      <property name="text">
       <string>Среднее время запусков</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="averageTime">
      <property name="text">
       <string>time</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="averageTimeBestL">
      <property name="text">
       <string>Среднее лучшее время</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="averageTimeBest">
      <property name="text">
       <string>time</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="bestResultL">
      <property name="text">
       <string>Лучший рейтинг</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="bestResult">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="averageResultL">
      <property name="text">
       <string>Средний рейтинг</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="averageResult">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="countConvolutionL">
      <property name="text">
       <string>Количество сверток</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="countConvolution">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="launchNumberL">
      <property name="text">
       <string>Номер запуска</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QComboBox" name="launchNumber"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="subLaunchNumberL">
      <property name="text">
       <string>Номер субЗапуска</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QComboBox" name="subLaunchNumber"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="subAverageResultL">
      <property name="text">
       <string>Средний рейтинг СЗ</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="subAverageResult">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="subBestResultL">
      <property name="text">
       <string>Лучший рейтинг СЗ</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLabel" name="subBestResult">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="subCountConvolutionL">
      <property name="text">
       <string>Количество сверток</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="2">
     <widget class="QPushButton" name="show">
      <property name="styleSheet">
       <string notr="true">color:white;background-color:#00BFFF;font-size:12px</string>
      </property>
      <property name="text">
       <string>Показать свертку</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLabel" name="subCountConvolution">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="2">
     <widget class="QPushButton" name="showSettings">
      <property name="styleSheet">
       <string notr="true">color:white;background-color:#00BFFF;font-size:12px</string>
      </property>
      <property name="text">
       <string>Показать настройки</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ResultForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ResultForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
